course/src/isometry.cc

```diff
@@ -0,0 +1,115 @@
+#include <isometry.h>
+
+namespace ekumen {
+  namespace math {
+
+  const Vector3 Vector3::kUnitX(1.,0,0);
+  const Vector3 Vector3::kUnitY(0,1.,0);
+  const Vector3 Vector3::kUnitZ(0,0,1.);
+  const Vector3 Vector3::kZero(0,0,0);
+
+  Vector3 Vector3::operator + (const Vector3 & obj) const
+  {
```

Comment:
In google style the `{` should be at the en of the previous line. Use an auto-formatter.